### ISSUE TITLE
Ethos Suite 1.5 RC0 fails to start on windows 11

### ISSUE THREAD

**Issue by fdm225** [FrSkyRC/ETHOS-Feedback-Community#3259]
did a fresh install of ethos suite and upon starting it am getting the following:

![image](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/1608096/7026e357-c16b-4f11-b710-3f6d98b997b6)

When I look at the crashpad log I see the following:

Hid write error: 
Error: Stack decoding error
Error: Error: Cannot parse given Error object

Hid write error: 
Error: Stack decoding error
Error: Error: Cannot parse given Error object

Hid write error: 
Error: Stack decoding error
Error: Error: Cannot parse given Error object

Hid write error: 
Error: Stack decoding error
Error: Error: Cannot parse given Error object

Hid write error: 
Error: Stack decoding error
Error: Error: Cannot parse given Error object

Hid write error: 
Error: Stack decoding error
Error: Error: Cannot parse given Error object

Hid write error: 
Error: Stack decoding error
Error: Error: Cannot parse given Error object

Hid write error: 
Error: Stack decoding error
Error: Error: Cannot parse given Error object

**Comment by Nicholas-Luoyi** [comment]
If you could close the error dialog. You can switch off "Popup dialog when a fatal error occurs" in Suite settings page. It should be OK.

**Comment by packetpilot** [comment]
suggestion for @fdm225: change "RC1" to "RC0" in the title (unless I'm just utterly confused and there _is an actual_ RC1)

tl;dr `sed s/RC1/RC0/`

**Comment by fdm225** [comment]
@packetpilot  ... title changed, thanks.

@Nicholas-Luoyi if I close the dialog it continues to pop up too fast to make any changes within the program.  I am happy to collect logs if needed, please let me know.

**Comment by Nicholas-Luoyi** [comment]
It seems that Suite is keeping writing HID data. Please remove all your frsky devices connection before you switch off the pop up option.

**Comment by fdm225** [comment]
@Nicholas-Luoyi by remove the connections are you saying to unplug the radio or something different?  I currently do not have the radio connected to the USB but am still getting the continual popup to the point where I cannot change the setting.

thanks,
dave

**Comment by Nicholas-Luoyi** [comment]
OK. Then it should be a bug. I will fix it.
